Sources/AppcuesKit/Data/Analytics/AnalyticsTracker.swift

```diff
@@ -117,7 +117,7 @@ internal class AnalyticsTracker: AnalyticsTracking, AnalyticsSubscribing {
     private func process(qualifyResponse: QualifyResponse, activity: Activity) {
         if let experienceRenderer = container?.resolve(ExperienceRendering.self) {
             let experiments = qualifyResponse.experiments ?? []
-            let qualifiedExperienceData: [ExperienceData] = qualifyResponse.experiences.map { experience in
+            let qualifiedExperienceData: [ExperienceData] = qualifyResponse.experiences.parsed().map { experience in
```

Comment:
Here is where we only take the properly parsed items currently, not the failures. In the future we can try to do something with the failed experiences to log flow issues.